vmaas/reposcan/repodata/updateinfo.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, filename):
                 update["type"] = elem.get("type")
                 update["id"] = text_strip(elem.find("id"))
                 update["title"] = text_strip(elem.find("title"))
+                # Is the false here good choice ? Shouldn't the default be true ?
+                update["reboot"] = False
```

Comment:
I think `False` should be default. But in SQL default is `true`, I'd set `false` there as well to be consistent.

Reply:
I see, however it'll result into incorrect reboot_suggested flags for most of advisories. We should find a way how to set it to correct value for all existing advisories in DB. Something like marking all repos revision to timestamp 0, so the reboot flag will be updated for all advisories in all repos (but I'm afraid we'd need to implement mechanism to update changed advisory metadata)

And second thing, I checked some updateinfo.xml and there is `reboot_suggested` flag for both advisory and packages in the advisory. Here we go through packages in advisory but we could just take the flag of the advisory and don't care about individual packages.

Reply:
What about something like this. Please check my updated `update_store.py` I first do an on-conflict update, and then lookup of values for returning. We do the same in patchman-engine on many places and it seems to be the simplest way for insertin/updating and getting all referneced rows.